modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/PublicationsDisplayContext.java

```diff
@@ -455,7 +455,7 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 				).buildString(),
 				"times-circle", "delete",
 				_language.get(_httpServletRequest, "delete"), "get", "delete",
-				null),
+				"async"),
```

Comment:
This reverts your fix in https://github.com/liferay/liferay-portal/commit/61e69aa8dac8bc4690a5cb012b54c96c222c0587 .

Please resend with a solution that takes care of both issues.